documentation/docs/scenarios/ui5-sap-cp/Readme.md

```diff
@@ -1,9 +1,6 @@
-# Create a Pipeline for SAP UI5 or SAP Fiori on SAP Cloud Platform
-
-Create an application based on SAP UI5 or SAP Fiori and deploy the build result into an SAP Cloud Platform account in the Neo environment.
-
-This document describes a scenario step, which means that it combines various different steps to create a complete pipeline.
+# Build and Deploy SAP UI5 or SAP Fiori Applications on SAP Cloud Platform with Jenkins
```

Comment:
> with Jenkins

Not sure if it required to mention Jenkins here explicitly. "With Jenkins" is already implied by using the piper lib.

Reply:
My intention was to enhance the findability and make the heading more specific so that I can use it as a link title when linking from the Learning Journey (in which the Jenkins usage is not obvious), for example.

Reply:
OK, here we have - like very often in life - different targets which competes with each other. From my point of view both is fine (having Jenkins here and not having Jenkins here ...)